src/components/admin/settings/general/Update.vue

```diff
@@ -1,5 +1,7 @@
 <script setup lang="ts">
+import { ref, watchEffect } from 'vue';
```

Comment:
remove this import, nuxt auto imports it

Reply:
ok